src/index.ts

```diff
@@ -19,7 +22,6 @@ export {
     RetryOptions,
     DummyOrchestrationContext,
     DummyEntityContext,
-    app,
-    input,
-    trigger,
+    createOrchestrator,

```

Comment:
I don't think we were exporting `createOrchestrator` and `createEntityFunction` from the package before this PR. Pretty sure they're just used for internal testing, right?

Reply:
You're right, I removed them. They were exported from `shim` but not from `index`. Also, I don't know what was going with the git history, it looks like my local `v3.x` branch that I branched out of was ahead from remote by 3 commits for some reason. Anyway, I rebased and should be fixed now.